api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,27 @@
+# Multi cloud API Automation
+
+**Introduction**
+
+This framework has developed in [JUNIT](https://junit.org/) using [Gradle](https://gradle.org/) Build tool.
+Junit framework details [Link](https://www.tutorialspoint.com/junit/index.htm)
+
+**Why this?**
+
+Junit is widely used testing framework along with Java Programming Language. You can use this automation framework for both unit testing and UI testing.
+
+**How to install and use?**
+
+ Download Intellij idea community edition tool and install in your pc.
```

Comment:
Is this dependent on IDE, Without IDE also It should be able to work.
May be You can give reference for the Dev-Setup for a gradle project like this to consider as an example.
[https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html](url)

Reply:
Yes. we can add this Link.

Reply:
I have changed. Please review.